### PR TITLE
BB-7: fix w/8.2 merge

### PR DESCRIPTION
**Pull request by alexanderchan-scality**

### CODE REVIEW

File: extensions/lifecycle/objectProcessor/task.js [hunk 1]
```diff
@@ -3,14 +3,15 @@
 const async = require('async');
 const werelogs = require('werelogs');
 const { errors } = require('arsenal');
-const {
-    DEFAULT_LIVE_ROUTE,
-    DEFAULT_READY_ROUTE,
-} = require('arsenal').network.probe.ProbeServer;
+// const {
```
Comment (jonathan-gramain) [inline extensions/lifecycle/objectProcessor/task.js:6]: Are the probe-related changes wanted? Just checking but good for me if that's the case

**Review by jonathan-gramain — APPROVED** [review]

File: extensions/lifecycle/objectProcessor/task.js (hunk 1 above)
Comment (alexanderchan-scality) [reply extensions/lifecycle/objectProcessor/task.js:6]: yes. I believe the plan is to move to the generic `ProbeServer` over the `HealthProbeServer`.
atm, `healthProbeServer` seems to also handle metrics which deviates from its original purpose.

**Review by anurag4DSB — APPROVED** [review]

**Pull request merged**